### PR TITLE
Update south-african-journal-of-animal-science.csl

### PR DESCRIPTION
**Pull request by rmzelle**
change and "text" to "symbol"
change name as sort order from first to always

### CODE REVIEW

File: south-african-journal-of-animal-science.csl [hunk 1]
```diff
@@ -39,7 +39,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name delimiter=", " and="text" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="first"/>
+      <name delimiter=", " and="symbol" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="always"/>
```
Comment (rmzelle) [inline south-african-journal-of-animal-science.csl]: @Yusufras, `name-as-sort-order="always"` is not valid CSL. It should be `name-as-sort-order="all"` (see the description of "name-as-sort-order" under http://docs.citationstyles.org/en/stable/specification.html#name). Could you correct this?

Comment (rmzelle) [inline south-african-journal-of-animal-science.csl]: (you can update this pull request by using the edit icon at https://github.com/Yusufras/styles/blob/patch-16/south-african-journal-of-animal-science.csl)

**Comment by Yusufras** [comment]
The changes made does not seem to work, although i followed the process.

**Comment by rmzelle** [comment]
> The changes made does not seem to work, although i followed the process.

No, you were fine. I've just been too busy to take a look at this.

**Comment by rmzelle** [comment]
All done!

**Pull request merged**